### DISCUSSION TITLE
Draft Front Matter Causing Issues

### DISCUSSION THREAD

**Discussion by trevor-moore-dev** [facebook/docusaurus#10653] [Q&A]
As soon as I set `draft: true` on a markdown file inside a deeply nested directory of a multi-instance plugin-content-docs, I get the following error:

```
[ERROR] MDX loader can't read MDX metadata file "/../.docusaurus/docusaurus-plugin-content-docs/../test.json". Maybe the isMDXPartial option function was not provided?

[ERROR] [Error: ENOENT: no such file or directory, open '/../.docusaurus/docusaurus-plugin-content-docs/../test.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/../.docusaurus/docusaurus-plugin-content-docs/../test.json'
}
```

Any thoughts on what the problem could be here?

**Comment by slorber** [comment]
It's impossible to help you the way you report this. Please provide a runnable docusaurus.new repro with the precise file to modify to get this error.

![CleanShot 2024-11-08 at 11 23 49](https://github.com/user-attachments/assets/a5444991-902b-4236-a992-67db20417cf8)
